python/MooseDocs/extensions/tagging.py

```diff
@@ -98,6 +98,22 @@ def postExecute(self):
                 for entry in key_list_regex:
                     regex_replace=f"'{entry}':"
                     tag_dict_str=re.sub(regex_replace,entry+':', tag_dict_str)
+                    # add relative link from built from the path at the end of the tagging entry
+                    if (entry == 'path'):
+                        path_value = tag_dict_str.split("path: ")[1].split(',')[0]
+
+                        # Split at the first modules/ (usually moose/modules/)
+                        if 'modules/' in path_value:
+                            path_value = path_value.split('modules/', 1)[1]
+                        else:
+                            # splitting at content is dangerous, same named paged within two different modules/xxx/doc/content
+                            # could be mixed together
+                            path_value = path_value.split('content/')[1]
```

Comment:
To make this less dangerous, we might try using `self.translator.findPages` here to check for possible page duplicates. See `Translator.py:151`.

Reply:
lemme try

Reply:
umm findPages isnt working great. Not finding path_value and not finding its original pre-cut location. 

content/ will work great for the VTB, and modules/ will work great for modules pages (unless they refer to other modules)

Reply:
Sounds good - just wanted to see if that made things more robust. Thanks for trying!